content/docs/getting-started/app-maintenance.md

```diff
@@ -22,3 +22,21 @@ Whenever you deploy an app update with `cf push`, your app will use the latest b
 ## Avoiding downtime during buildpack or app updates
 **Note that your app may be unavailable while a `cf restage` or `cf push` is in progress.** Avoid this by [setting up zero-downtime deployments for your app](https://cloud.gov/docs/apps/production-ready/#zero-downtime-deploy), so that every new version of your app picks up the latest buildpack without impacting your app's availability.
 
+## Operating system patching
+
+The cloud.gov platform team is responsible for maintenance and patching of the operating system (Ubuntu) and other platform components. Your applications benefit from these updates automatically.
+
+How this works:
+
+1. The Ubuntu team releases an update.
+1. The [Cloud Foundry BOSH](http://bosh.cloudfoundry.org/) team packages the Ubuntu update into a [“stemcell” image](https://bosh.cloudfoundry.org/docs/stemcell.html) and releases the stemcell update, typically within 24-48 hours.
+1. The cloud.gov system picks up the stemcell update, checks it with tests, and deploys the update to production.
```

Comment:
Nit: we could be a little more specific about tests here. What about something like "The cloud.gov system picks up the stemcell update, deploys the new stemcell to our staging environment, and verifies that all components are operating as expected before deploying to production."?